### ISSUE TITLE
PHP8 / Contao 4.13 LTS

### ISSUE THREAD

**Issue by trilobit-gmbh** [trilobit-gmbh/contao-zipuploads-bundle#2]

**Comment by operatorone** [comment]
Any news on this?

**Comment by trilobit-gmbh** [comment]
Ja, wir planen unsere Bundles fit für PHP 8 und die nächste LTS-Version von Contao (4.13) zu machen.
Wenn Du Zeit und Lust hast freuen wir uns über einen Pull-Request hierzu ;-)

Leider kann ich im Moment aber nicht sagen, wie schnell wir das hin bekommen. Die Umsetzung wird in der Regel sein in

freien Zeitslots ;-(
wenn wir es selbst in einer neueren Version benötigen
wenn es einen Pull-Request gibt ;-)
wenn wir dafür bezahlt werden ;-)
Viele Grüße
Peter

**Comment by operatorone** [comment]
Hallo Peter,

wie hoch wäre der Aufwand, die Anpassungen für PHP8 durchzuführen? Leider kann ich dir mangels Expertise hier nicht so viel beisteuern, aber ich bin gerne beim Testen dabei.

Viele Grüße
Tilman

**Comment by trilobit-gmbh** [comment]
Kommt noch vor meinem Urlaub; ist gerade im internen Testing und Freigabe ;-)
Kleine Spende willkommen wenn Dir die Erweiterung gefällt und Du sie nutzt ;-); z. B. über den Button am Ende dieser Seite (https://www.trilobit.de/trilobits/das-contao-calculator-bundle.html) - wir haben keine Extra Seite für dieses Bundle; aber das kommt auch an ;-)
